sql/sql_connect.cc

```diff
@@ -255,6 +262,105 @@ void time_out_user_resource_limits(THD *thd, USER_CONN *uc)
   DBUG_VOID_RETURN;
 }
 
+static uint64 get_connection_delay_time(uint32 count)
+{
+  /*
+   * Based on that invalid password errors will cause user can't connect server
+  ever and server close connection if it got message from client after
+  wait_timeout
+   * so,
+   step = read_timeout/(max_password_errors - password_errors_before_delay)
+  delay = step *(failed_count  -password_errors_before_delay)
```

Comment:
either call it `count` here or call it `failed_count` in the function definition. I mean, the comment should better match the code.